analytics.go

```diff
@@ -9,6 +9,7 @@ type AnalyticsRequest struct {
 	AggregateUnit   string           `json:"aggregate_unit,omitempty"`
 	TimeZone        string           `json:"time_zone,omitempty"`
 }
+
 type AnalyticsResponse struct {
 	Data            []AnalyticsData  `json:"data,omitempty"`
 	AnalyticsFilter *AnalyticsFilter `json:"filters,omitempty"`
```

Comment:
Apologies for asking this of you since this wasn't your change, but your PR made me look at things again and I am now realizing I missed this in the original PR. Would you rename this field in your PR to just be `Filters`. We've not released this yet, and so I'd rather we get this done right versus letting it bleed out once we do release `v1.4.0`.

Reply:
`AnalyticsFilter` has been changed to `Filters` 👍

Reply:
I think you may have misread my comment, I'm sorry. Was only wanting the field name updated, not the type too. The type made sense because it's the filters for the Analytics stuff, but `AnalyticsResponse.AnalyticsFilters` stutters a little bit, while `AnalyticsResponse.Filters` is easier to communicate.

Reply:
I did misread your comment. Apologies 😅 I went ahead and changed that in the new commit. Please let me know if you see anything else 👍